app/views/home/index.html.erb

```diff
@@ -0,0 +1,7 @@
+<div class="row">
+<% @photos_low_res.find_each do |photo| %>
```

Comment:
In fact, if you look to Rails source code, `.find_each` with no arguments calls `.each` internally: https://github.com/rails/rails/blob/5473e390d362755125d2f47b64ef0a135f2fe111/activerecord/lib/active_record/relation/batches.rb#L63